src/main/resources/sql/grade_creation.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE grade (
```

Comment:
rename to grade_post because in future we will have grade for something another. and it is more clearly name